dev/com.ibm.ws.jdbc_fat_krb5/fat/src/com/ibm/ws/jdbc/fat/krb5/containers/OracleKerberosContainer.java

```diff
@@ -13,39 +13,27 @@
 import static com.ibm.ws.jdbc.fat.krb5.containers.KerberosContainer.KRB5_KDC;
 import static com.ibm.ws.jdbc.fat.krb5.containers.KerberosContainer.KRB5_REALM;
 
-import java.io.FileInputStream;
-import java.io.FileWriter;
-import java.io.IOException;
-import java.nio.file.Files;
-import java.nio.file.Path;
-import java.nio.file.Paths;
-import java.time.Duration;
-import java.util.Properties;
-
 import org.testcontainers.containers.Network;
 import org.testcontainers.containers.OracleContainer;
-import org.testcontainers.containers.output.OutputFrame;
-import org.testcontainers.containers.wait.strategy.LogMessageWaitStrategy;
-
-import com.ibm.websphere.simplicity.log.Log;
-import com.ibm.ws.jdbc.fat.krb5.FATSuite;
+import org.testcontainers.utility.DockerImageName;
 
-import componenttest.custom.junit.runner.FATRunner;
+import componenttest.containers.SimpleLogConsumer;
 
 public class OracleKerberosContainer extends OracleContainer {
 
     private static final Class<?> c = OracleKerberosContainer.class;
-    private static final Path reuseCache = Paths.get("..", "..", "cache", "oracle.properties");
-    // NOTE: If this is ever updated, don't forget to push to docker hub, but DO NOT overwrite existing versions
-    private static final String IMAGE = "kyleaure/krb5-oracle:2.0";
 
-    private boolean reused = false;
-    private String reused_hostname;
-    private int reused_port;
+    // NOTE: If this is ever updated, don't forget to push to docker hub, but DO NOT overwrite existing versions
+    //TODO update this image to be built on top of gvenzl/oracle-xe
+    private static final String IMAGE_NAME_STRING = "kyleaure/oracle-18.4.0-expanded:1.0.full.krb5";
+    private static final DockerImageName IMAGE_NAME = DockerImageName.parse(IMAGE_NAME_STRING).asCompatibleSubstituteFor("gvenzl/oracle-xe");
```

Comment:
Similar to previous comment.  Consider externalizing these two values (via public) so they can be used instead of hardcoded values in other classes.